wagtail/tests/demosite/migrations/0001_initial.py

```diff
@@ -105,7 +106,7 @@ class Migration(migrations.Migration):
             name='ContactPage',
             fields=[
                 ('telephone', models.CharField(blank=True, max_length=20)),
-                ('email', models.EmailField(blank=True, max_length=75)),
+                ('email', models.EmailField(blank=True, max_length=(254 if django.VERSION >= (1, 8) else 75))),
```

Comment:
We could, providing we also set it explicitly in models.py (so that Dj1.7 won't see a difference and report a missing migration) - didn't feel like there was much to choose between these options.